docs/how-to-guides.md

```diff
@@ -537,14 +537,15 @@ v3, but node extensions (`%bs.*`) are not.
 #### `@bs` attribute becomes `@u`
 
 The `@bs` attribute, used for uncurried application (see the ["Binding to
-callbacks" section](./communicate-with-javascript.md#binding-to-callbacks)),
-becomes `@u`.
+callbacks"
+section](/working-with-js-objects-and-values.md#binding-to-callbacks)), becomes
+`@u`.
 
 #### `@bs.val` is gone
 
 The `@bs.val` attribute is no longer necessary, and can be removed from
 `external` definitions. See more information in the ["Using global functions or
-values"](./communicate-with-javascript.md#using-global-functions-or-values)
+values"](/working-with-js-objects-and-values.md#using-global-functions-or-values)
```

Comment:
@jchavarri what's the convention for links? (is there one?) 

Should I:

- Add `.md` at the end of page names?
- Add `/`, `./`, or nothing at the beginning of links?

Reply:
I don't think there's one. Maybe we could keep the `.md` at the end (i am not sure if vitepress works without that), and relative paths `./`.

Reply:
Made all the links between pages use `./` and `.md` in https://github.com/melange-re/melange-re.github.io/pull/208/commits/c07670105875f978ed75849d2b36da47130343bc